TESTS/storage_abstraction/basicAPI/basicAPI.cpp

```diff
@@ -371,17 +377,16 @@ control_t test_programDataUsingProgramUnit(const size_t call_count)
     TEST_ASSERT(rc >= 0);
     if (rc == ARM_DRIVER_OK) {
         TEST_ASSERT_EQUAL(1, capabilities.asynchronous_ops);
-        return (call_count < REPEAT_INSTANCES) ? CaseTimeout(200) + CaseRepeatAll: CaseTimeout(200);
+        return (call_count < REPEAT_INSTANCES) ? CaseTimeout(ERASE_CASE_TIMEOUT) + CaseRepeatAll: CaseTimeout(ERASE_CASE_TIMEOUT);
```

Comment:
You've combined two different things into a single commit. It is better to separate conceptually different changes into distinct commits. 

What is the justification for changing the erase Timeouts? Were you hitting this timeout on the ST NVM?